app/flux/actions/AppActionCreators.js

```diff
@@ -0,0 +1,39 @@
+var ActionTypes = require('../constants/TrainifyConstants').ActionTypes;
+var Dispatcher = require('../dispatcher/TrainifyAppDispatcher');
+
+var AppStore = require('../stores/AppStore');
+var UserAPI = require('../API/UserAPI');
+
+module.exports = {
+
+    /**
+     * The navigation menu has been clicked, 
+     *
+     */
+    clickNavigationMenu: () => {
```

Comment:
I don't understand why this is even here, this is a simple UI menu action that could be handled from within the view.